### DISCUSSION TITLE
Seven Seas Protocol

### DISCUSSION THREAD

**Discussion by markuspluna** [stellar/sorobanathon#92] [General]
Wrote a pirate-themed OHM fork for fun, has some neat mechanisms that might be useful in actually valuable DeFi protocols 

Codebase and writeup here:
https://github.com/markuspluna/seven-seas

**Comment by achallagundla** [comment]
Lol this is so cool! Love this submission, very creative!!

**Comment by achallagundla** [comment]
Heya @markuspluna  ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 2000 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
